events/admin.py

```diff
@@ -38,14 +38,22 @@ class EventAttendeesFormInline(OrderableAdmin, admin.TabularInline):
     fk_name = 'event'
     extra = 0
     list_editable = ('user', 'email', 'preferences')
-    readonly_fields = ('time_registered',)
-    fields = ('attendee_nr', 'user', 'email', 'anonymous', 'preferences', 'time_registered')
     formfield_overrides = {
         JSONField: {'widget': PrettyJSONWidget()}
     }
     can_delete = True
     ordering = ['attendee_nr']
 
+    def get_fields(self, request, event):
+        fields = ['attendee_nr', 'user', 'email', 'anonymous', 'preferences', 'time_registered']
+        if event and event.sign_up_avec:
+            fields.append('avec_for')
+        return fields
+    def get_readonly_fields(self, request, event):
+        readonly_fields = ['time_registered']
+        if event and event.sign_up_avec:
```

Comment:
Thought that would make more sense as well, But apparently it's required since readonly fields has to be included in normal fields to be displayed.
https://docs.djangoproject.com/en/3.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.readonly_fields

I made it readonly because I didn't really see any reason to edit it, since people not using avec will use bordssällskap. But you're right, there will probably be cases where we want to edit it so I'll fix it